website/docs/about/contribution.md

```diff
@@ -13,8 +13,8 @@ At this time we are not accepting external contributions from non-HashiCorp empl
 
 While we don’t accept direct contributions of new components or patterns, we do welcome contributions in the following forms:
 
-* [Reporting](https://github.com/hashicorp/design-system/issues/new/choose) or [fixing](#code-contributions) bugs and [giving feedback](https://docs.google.com/forms/d/e/1FAIpQLSdPShbSiUYgimppZeqWT99bVi6zLbKDThQUQTirH3RCj2okdg/viewform)
-* [Proposing new components or patterns](https://docs.google.com/forms/d/e/1FAIpQLScpMXgrUTVT5fYriu4Pp48r4Nl_eCPluVnJLg0Yg3NXsRWvIA/viewform)
+* [Reporting](support#file-an-issue) or [fixing](#code-contributions) bugs and [giving feedback](https://docs.google.com/forms/d/e/1FAIpQLSdPShbSiUYgimppZeqWT99bVi6zLbKDThQUQTirH3RCj2okdg/viewform)
+* [Proposing new components or patterns](about/support#request-a-feature)
 * [Adding an icon](#icon-contributions)
```

Comment:
I ran into the same issue with the skip link, where the page isn't long enough for the proper jump. 

```suggestion
* Adding an icon (see [Icon contributions](#icon-contributions))
```